### ISSUE TITLE
Six pipeline failures

### ISSUE THREAD

**Issue by ahrtr** [etcd-io/etcd#13944]
I just submitted a test [PR](https://github.com/etcd-io/etcd/pull/13943) (no any code change at all), there are still 6 pipeline failures.  We can ignore the code scan failure for now. The other 5 failures are caused by PR [13888](https://github.com/etcd-io/etcd/pull/13888), which was **NOT** green when it's merged. 

Proposed action for now: rollback the PR [13888](https://github.com/etcd-io/etcd/pull/13888) for now so as to unblock all new PRs against main. We can bump the opentelemetry later in a separate PR. 

cc @serathius @ptabor @spzala  for opinion.

**Comment by spzala** [comment]
That sounds good. Thanks @ahrtr

**Comment by ahrtr** [comment]
Closing this issue since the PR [13945](https://github.com/etcd-io/etcd/pull/13945) has already been merged.
